Cargo.toml

```diff
@@ -20,18 +20,20 @@ repository.workspace = true
 [lib]
 crate-type = ["cdylib"]
 
-# fields to configure build with WASM reproducibility, according to specs  
-# in https://github.com/near/NEPs/blob/master/neps/nep-0330.md 
+# fields to configure build with WASM reproducibility, according to specs
+# in https://github.com/near/NEPs/blob/master/neps/nep-0330.md
 [package.metadata.near.reproducible_build]
 # docker image, descriptor of build environment
-image = "sourcescan/cargo-near:0.8.2-rust-1.81.0"
+image = "sourcescan/cargo-near:0.13.2-rust-1.84.0"
 # tag after colon above serves only descriptive purpose; image is identified by digest
-image_digest = "sha256:3b5e6efa1d0cef710009f9cc20c845d316187cc170455edc5ba7fdf471c19655"
-# build command inside of docker container 
-# if docker image from default gallery is used https://hub.docker.com/r/sourcescan/cargo-near/tags,
-# the command may be any combination of flags of `cargo-near`,
-# supported by respective version of binary inside the container besides `--no-locked` flag
-container_build_command = ["cargo", "near", "build"]
+image_digest = "sha256:b41cb89907f92b114da9e2be4e109bff30ab792c22ddcbd2a4cff8e340cb9acb"
+container_build_command = [
+    "cargo",
+    "near",
+    "build",
+    "non-reproducible-wasm",
+    "--locked",
+]
```

Comment:
same blocks with same image can be added to `disucssions` and `community` manifests, as in [commit](https://github.com/dj8yfo/neardevhub-contract/commit/01ed71a645a553b0c566acf7dc4339930b80de67), 
to allow nearblocks+sourcescan verification of any new subcontracts , deployed from community-factory, 
which currently runs `cargo near build` as verification implementation.

First `community-factory` would've to be redeployed , though, as https://nearblocks.io/address/community.devhub.near?tab=contract shows it hasn't been deployed after #151 merge